phpstan.neon.dist

```diff
@@ -0,0 +1,18 @@
+includes:
+	- vendor/phpstan/phpstan-phpunit/extension.neon
```

Comment:
missing `- vendor/ekino/phpstan-banned-code/extension.neon`

Reply:
i think we have an issue into ekino/phpstan-banned-code. Parameters are not merged but added for node types that are not FuncCall. Then you can disable a check on a node type that is not FuncCall if you load `vendor/phpstan/phpstan-phpunit/extension.neon`

Reply:
Because I think your config is not up to date with last changes, see https://github.com/ekino/phpstan-banned-code#advanced-usage

Reply:
Issue is `Expr_FuncCall` and `Stmt_*|Expr_*` are handled diffenrently. `Expr_FuncCall` can be overridden but not `Stmt_*|Expr_*`. The merge of banned code still make appear `Stmt_Echo` that we want to exclude from banned code for this lib.